gridfs/gridfs.c

```diff
@@ -70,7 +70,7 @@ typedef struct {
 	struct stat sb;
 } php_stdio_stream_data;
 
-static int setup_file_fields(zval *zfile, char *filename, int size TSRMLS_DC);
+static int setup_file_fields(zval *zfile, char *filename, long size TSRMLS_DC);
```

Comment:
This can potentially also be a size_t instead. Should be the same as below though.